Sequential index creation (#600)
gentics/mesh: CHANGELOG.adoc

icon:check[] GraphQL: Resolved links will now contain the hostname of the target branch, not the latest branch.

> icon:check[] Elasticsearch: Search indices are now sequentially. This was causing problems when many indices would have to be created.

Reviewer: ```suggestion
icon:check[] Elasticsearch: Search indices are now created sequentially. Parallel creation of indices caused problems when many indices would have to be created.
```
